pennylane/labs/resource_estimation/resource_operator.py

```diff
@@ -328,43 +302,33 @@ def default_pow_resource_decomp(cls, pow_z: int, *args, **kwargs) -> list:
         """
         raise ResourcesNotDefined
 
-    @classmethod
-    def pow_resource_decomp(cls, pow_z, *args, **kwargs) -> list:
-        r"""Returns a list representing the resources for an operator
-        raised to a power.
-
-        Args:
-            pow_z (int): exponent that the operator is being raised to
-        """
-        return cls.default_pow_resource_decomp(pow_z, *args, **kwargs)
-
     @classmethod
     def set_resources(cls, new_func: Callable, override_type: str = "base"):
         """Set a custom function to override the default resource decomposition.
 
-        This method allows users to replace any of the `resource_decomp`, `adjoint_resource_decomp`,
-        `ctrl_resource_decomp`, or `pow_resource_decomp` methods globally for every instance of
+        This method allows users to replace any of the `default_resource_decomp`, `default_adjoint_resource_decomp`,
+        `ctrl_resource_decomp`, or `default_pow_resource_decomp` methods globally for every instance of
         the class.
 
         """
         if override_type == "base":
             keys = cls.resource_keys.union({"kwargs"})
             _validate_signature(new_func, keys)
-            cls.resource_decomp = new_func
+            cls.default_resource_decomp = new_func
```

Comment:
I thought we're getting rid of the behaviour of mutating class definitions?

Reply:
Yes, this is just cause I used ctrl f and replace all probably.